provider/parameter.go

```diff
@@ -6,7 +6,7 @@ import (
 	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"net/url"
+	"github.com/coder/terraform-provider-coder/v2/provider/helpers"
```

Comment:
nit: ordering

Reply:
Addressed in https://github.com/coder/terraform-provider-coder/pull/421/commits/de3cf097faa71bf1512cfb325e9d4dd39d0999a2